packages/components/src/Chips/InternalChipDismissible/hooks/useInternalChipDismissible.ts

```diff
@@ -21,23 +20,55 @@ export function useInternalChipDismissible({
   const availableChipOptions = chipOptions.filter(
     chip => !selected.includes(chip.value),
   );
-  const { liveAnnounce } = useLiveAnnounce();
 
   const actions = {
-    handleChipRemove: (chip: typeof chipOptions[number]) => {
-      return () => {
-        liveAnnounce(`${chip.label} Removed`);
-        onChange(selected.filter(val => val !== chip.value));
-      };
+    handleChipRemove: (value: string) => {
+      return () => onChange(selected.filter(val => val !== value));
     },
+
     handleChipAdd: (value: string) => onChange([...selected, value]),
     handleCustomAdd: onCustomAdd,
+
     handleChipClick: (value: string) => {
       if (onClick === undefined) return;
       return (event: MouseEvent<HTMLButtonElement>) => onClick(event, value);
     },
+
     handleEmptyBackspace: () => {
-      actions.handleChipRemove(sortedVisibleChipOptions[selected.length - 1])();
+      actions.handleChipRemove(selected[selected.length - 1])();
+    },
+
+    handleWrapperKeyDown: (event: KeyboardEvent<HTMLDivElement>) => {
+      const target = event.target;
+      if (!(target instanceof HTMLElement)) return;
+      if (target instanceof HTMLInputElement && target.value) return;
```

Comment:
why are we doing this?

Reply:
Totally a good question.  Keeping me on my toes! 😬 

The first one is that there is that there is an error when we try to access previous/next element sibling when we don't do the checks on the first highlighted line.

The second highlighted line there is to prevent using the left/right arrow keys when you start typing when there is an input.  I am assuming that Darryl and Chris worked out that functionality in a previous review and that was planned.

Reply:
Resolving, but feel free to unresolve if this answer doesn't make you happy!!!

Reply:
For the second line, can we also make that into a self explanatory variable or add a comment? First preferred, but it not possible add comment.

For the first one, I guess my confusion is around why we need this escape hatch. I don't mean this from a TS perspective, which I get. Its more that if we didn't have a HTMLElement to begin with, then we would not attach this method to it in the first place.

Reply:
@timoteialbu So for the first check, since the event is a a keyboard event on an html element you're thinking we don't really need that check to begin with?

Would `const target = event.target as HTMLElement;` and then removing the check make sense?

Reply:
I don't like forcing a cast like that just because this will hide future states where that type will change. I found something that we can do like this:

```
handleWrapperKeyDown: (
      event: KeyboardEvent<HTMLDivElement> & {
        target: HTMLDivElement;
      },
    ) => {
```

Reply:
Damn okay! That makes sense!!